docs/sources/flow/setup/start-agent.md

```diff
@@ -172,4 +172,41 @@ Replace the following:
 * `BINARY_PATH`: The path to the Grafana Agent binary file
 * `CONFIG_FILE`: The path to the Grafana Agent configuration file.
 
+### Set up Grafana Agent as a Linux systemd service
+
+You can set up and manage the standalone binary for Grafana Agent as a Linux systemd service.
+
+1. To create a new user called `grafana-agent` run the following command in a terminal window:
+
+   ```shell
+   sudo useradd --no-create-home --shell /bin/false grafana-agent
+   ```
+
+1. Create a file in `/etc/systemd/system` called `grafana-agent.service` with the following contents:
```

Comment:
Should this be called `grafana-agent-flow.service` to align with what the system packages install?